test/packages/parallel/sql_input/fields/input.yml

```diff
@@ -0,0 +1,20 @@
+- name: input.name
+  type: constant_keyword
+  description: Sample field to be added.
+  value: sql
+- name: sql.metrics.numeric.*
+  type: double
+- name: sql.metrics.string.*
+  type: keyword
+- name: data_stream.dataset
+  external: ecs
+- name: data_stream.namespace
+  external: ecs
+- name: data_stream.type
+  external: ecs
+- name: service.address
+  external: ecs
```

Comment:
Should these four field external definitions moved to `ecs.yml` ?

Reply:
I don't think there is any preference about this. As you prefer.

Reply:
Ok! if there are no preferences. Let's keep it here, no need to change it :+1: